docs/pbc/all/search/algolia.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Algolia
+description: Algolia is a search engine that 
+---
+
+By default, all Spryker stores have [Elasticsearch](https://www.elastic.co/elasticsearch/) as their search engine. However, you can replace it with the Algolia one.
+
+The Algolia search engine that stands out from others due to its performance. With the Algolia app, your users can conduct advanced search of active concrete products in your store. 
+
+{% info_block infoBox "Product offers and product reviews" %}
+
+Algolia searches in product offers and product reviews as well. However, these entities are represented in product attributes in the Algolia search results.
+
+{% endinfo_block %}
+
+To use Algolia as your search engine, you need an account with Algolia. For details about Algolia integration, see [Integrate Algolia](/docs/pbc/all/search/integrate-algolia.html).
+
+## Searchable attributes
+
+Your users can search for active concrete products by the following attributes:
+
+- SKU
+- Name
+- Description
+- Keywords
+- Currency
+- Gross Price
+- Net Price
+- Product Abstract SKU
+- Rating (average approved rating for a respective abstract product)
+- Images
+- Categories
+- Attributes
+- Merchants
```

Comment:
Marketplace data structure is not supported, so I would prefer to remove this line.